dao/mongo.go

```diff
@@ -24,6 +24,7 @@ const (
 	bulkRefundStatus             = "bulk_refunds.status"
 	dataProviderID               = "data.provider_id"
 	externalPaymentTransactionID = "external_payment_transaction_id"
+	RefundsBatchSize             = 20
```

Comment:
This value should come from environment configs

Reply:
Moved to environment variable